src/bootstrap/src/core/build_steps/test.rs

```diff
@@ -1397,6 +1398,53 @@ impl Step for RunMakeSupport {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+pub struct RunMakeSupportTest {
```

Comment:
We use this naming format for crate tests to avoid confusions with other test suites.

```suggestion
pub struct CrateRunMakeSupport {
```

Reply:
Okay.